test/test_numerizer.rb

```diff
@@ -127,4 +131,42 @@ def test_ordinal_strings
     end
   end
 
-end
+  def test_ambiguous_cases
+    # Quarter ( Coin ) is Untested
+    # Second ( Time / Verb ) is Untested
+    assert_equal 'the 4th', Numerizer.numerize('the fourth')
+    assert_equal '1/3 of', Numerizer.numerize('a third of')
+    assert_equal '4th', Numerizer.numerize('fourth')
+    assert_equal '2nd', Numerizer.numerize('second')
+    assert_equal 'I quarter', Numerizer.numerize('I quarter')
+    assert_equal 'You quarter', Numerizer.numerize('You quarter')
+    assert_equal 'I want to quarter', Numerizer.numerize('I want to quarter')
+    assert_equal 'the 1st 1/4', Numerizer.numerize('the first quarter')
+    assert_equal '1/4 pound of beef', Numerizer.numerize('quarter pound of beef')
+    assert_equal 'the 2nd second', Numerizer.numerize('the second second')
+    assert_equal '1 second', Numerizer.numerize('one second')
+
+#   assert_equal 'I peel and quarter bananas', Numerizer.numerize('I peel and quarter bananas') # TODO: Find way to distinguish this verb
+  end
+
+  def test_ignore
+    assert_equal 'the second day of march', Numerizer.numerize('the second day of march', ignore: ['second'])
+    assert_equal 'quarter', Numerizer.numerize('quarter', ignore: ['quarter'])
+    assert_equal 'the five guys', Numerizer.numerize('the five guys', ignore: ['five'])
+    assert_equal 'the fifty 2', Numerizer.numerize('the fifty two', ignore: ['fifty'])
+  end
+
+  def test_bias_ordinal
+    assert_equal '4th', Numerizer.numerize('fourth', bias: :ordinal)
```

Comment:
Will `a fourth of` numerize to `a 4th of` when bias is ordinal?

Reply:
It should be added. Currently it results in "1 4th of" instead of "a 4th of".

Reply:
it'll result in "a 4th of" in the latest commit.